packages/babel-generator/src/buffer.js

```diff
@@ -14,6 +14,7 @@ export default class Buffer {
     this._indent = format.indent.base;
     this.format = format;
     this.buf = "";
+    this.last = "";
```

Comment:
interesting, thanks for explaining. I'm curious, in our case, how big is the difference?